sections/dashboard/FuturesPositionsTable/FuturesPositionsTable.tsx

```diff
@@ -277,16 +230,46 @@ const FuturesPositionsTable: FC<FuturesPositionTableProps> = ({
 									};
 									return (
 										<Currency.Price
-											currencyKey="sUSD"
 											price={cellProps.row.original.position.liquidationPrice}
-											sign="$"
-											conversionRate={1}
 											formatOptions={formatOptions}
 										/>
 									);
 								},
 								width: 115,
 							},
+							{
+								Header: (
+									<TableHeader>{t('dashboard.overview.futures-positions-table.pnl')}</TableHeader>
+								),
+								accessor: 'pnl',
+								Cell: (cellProps: CellProps<any>) => {
+									return (
+										<PnlContainer>
+											<ChangePercent value={cellProps.row.original.position.pnlPct} />
+											<div>
+												<Currency.Price price={cellProps.row.original.position.pnl} />
+											</div>
+										</PnlContainer>
+									);
+								},
+								width: 125,
+							},
+							{
+								Header: <TableHeader>Funding</TableHeader>,
+								accessor: 'funding',
+								Cell: () => {
+									return <div></div>;
```

Comment:
Are these left empty because we'll replace this with the other table in a separate PR?

Reply:
Something like that. That said. I can just copy it over instead of leaving it empty